From ce8ea2440943aeb8dfdb729d38981de27af6b67d Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Mon, 27 Jul 2020 21:21:04 -0700
Subject: [PATCH] arm64: defconfig: Explicitly disable CONFIG_ARM64_TLB_RANGE

This does not play well with LLVM's integrated assembler right now.

Link: https://github.com/ClangBuiltLinux/linux/issues/1106
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index e0f33826819f..98cea43368ec 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -70,6 +70,7 @@ CONFIG_KEXEC_FILE=y
 CONFIG_CRASH_DUMP=y
 CONFIG_XEN=y
 CONFIG_COMPAT=y
+# CONFIG_ARM64_TLB_RANGE is not set
 CONFIG_RANDOMIZE_BASE=y
 CONFIG_HIBERNATION=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
-- 
2.28.0.rc1

